### PR TITLE
Use modal dialog for videos (no more adds) + video cards for related videos

### PR DESCRIPTION
**Pull request by ge0ffrey**
Selection_826

### CODE REVIEW

**Comment by yurloc** [comment]
@ge0ffrey Can you make the dialog disappear on:
- hitting `Esc`,
- clicking outside of the dialog?

(Provided these can be achieved easily using a means available in Bootstrap.)

**Comment by yurloc** [comment]
Screen Shot 2021-09-27 at 15 17 44-fullpage

The card dimensions look wrong in the compatibility layout.

File: templates/macros.ftl [hunk 1]
```diff
@@ -202,22 +201,27 @@
     <#if relatedVideos?size &gt; 0>
         <h2>Related videos</h2>
         <#-- TODO use card layout after upgrade to Twitter bootstrap 5 and make it prettier (whitespace etc) -->
```
Comment (yurloc) [inline templates/macros.ftl:203]: Is this TODO still valid?

**Comment by ge0ffrey** [comment]
Good catches, I fixed all 3 comments :)

**Pull request merged**